lib/logo/api.ex

```diff
@@ -35,6 +35,36 @@ defmodule Logo.Api do
     draw_block(t1, 5, {2, 3})
     draw_block(t1, 6, {3, 3})
 
+    draw_block(e2, 0, {4, 1})
```

Comment:
Disse må bare være `e` (nå referer de til andre _e_-en.

Kan kanskje rename `e` til `e1` for symmetri.